src/api/eventsAPI.js

```diff
@@ -5,9 +5,12 @@ import { queryBuilder } from '../utils';
 const BASE_URL = 'https://resistance-calendar.herokuapp.com/v1/events';
 
 function getEvents({ page = 0, perPage = 25 }, filterParams) {
+  const pageUrl = `page=${page}`;
+  const perPageUrl = `per_page=${perPage}`;
+  const orderByUrl = '$orderby=start_date desc';
```

Comment:
Hmm I'd love to, but how would I do that?  It seems like that's in the realm of the server right?